infrastructure/src/main/java/mops/controllers/DozentEventController.java

```diff
@@ -88,6 +89,21 @@ public String handleFileUpload(@RequestParam("file") MultipartFile file,
     return "redirect:/feedback/dozenten/event/{veranstaltungsNr}";
   }
 
+  @PostMapping("/event/addStudent/{veranstaltungsNr}")
+  @RolesAllowed(ORGA_ROLE)
+  public String addStudent(@PathVariable Long veranstaltungsNr,
+                           RedirectAttributes redirectAttributes,
+                           String newStudent) {
+    Student student = new Student(newStudent);
+    Veranstaltung veranstaltung = veranstaltungen.getVeranstaltungById(veranstaltungsNr);
+    veranstaltung.addStudent(student);
+    veranstaltungen.save(veranstaltung);
+    redirectAttributes.addFlashAttribute("message", newStudent
+        + "wurde erfolgreich hinzugefügt!");
```

Comment:
Please put a whitespace before _wurde_. If you don't do it, there is no whitespace between username and _wurde_ in the UI.